src/sentry/notifications/notifications/organization_request/base.py

```diff
@@ -46,10 +46,13 @@ def build_notification_footer(self, recipient: Team | User) -> str:
         if isinstance(recipient, Team):
             raise NotImplementedError
 
-        # notification footer only used for Slack for now
         settings_url = self.get_settings_url(recipient, ExternalProviders.SLACK)
-        return self.role_based_recipient_strategy.build_notification_footer_from_settings_url(
-            settings_url, recipient
+        recipient_role_string = self.role_based_recipient_strategy.get_recipient_role_string(
+            recipient
+        )
+        return (
+            "You are receiving this notification because you're listed as an organization "
```

Comment:
@97varun It's still unfortunate we have to duplicat this text within getsentry, now we have two sources of truth for the same message. Can you fix?